src/Stratis.Bitcoin/Controllers/NodeController.cs

```diff
@@ -410,7 +410,7 @@ public async Task<IActionResult> GetTxOutAsync([FromQuery] string trxid, uint vo
         /// </remarks>
         /// <returns><see cref="OkResult"/></returns>
         [HttpPost]
-        [Route("shutdown-node")]
+        [Route("shutdown")]
```

Comment:
This is a change in the behaviour of the endpoint @dev0tion are you using this endpoint? does it effect you?

Reply:
It's going back to what it was before... thought as we weren't planning to change any of the endpoints, there wasn't any point with just one changing.